config.go

```diff
@@ -104,7 +104,8 @@ type config struct {
 	PGPass        string `long:"pgpass" description:"PostgreSQL DB password."`
 	PGHost        string `long:"pghost" description:"PostgreSQL server host:port or UNIX socket (e.g. /run/postgresql)."`
 	NoDevPrefetch bool   `long:"no-dev-prefetch" description:"Disable automatic dev fund balance query on new blocks. When true, the query will still be run on demand, but not automatically after new blocks are connected."`
-	SyncAndQuit   bool   `long:"sync-and-quit" description:"Sync to the best block and exit. Do not start the explorer or API."`
```

Comment:
I did away with `SyncAndQuit` since I made the sync part of the code to be among the last thing that is executed in the mainCore function in main.go. I found that part of the code to be redundant as most of the functions are already up and running by the time blockchain sync ends. This is [where](https://github.com/decred/dcrdata/pull/710/files#diff-7ddfb3e035b42cd70649cc33393fe32cR686) that piece of code would go but it if I put it back it would be pretty much redundant especially when no sync status page is running which will be common.

Reply:
ok. I could still get it back and we can try to reason with in place if we should keep it or not.

Reply:
Well we can discuss it now.  I may be missing something, as I don't get "redundant especially when no sync status page is running which will be common".  The `SyncAndQuit` flag was to *quit* dcrdata when sync is complete.  The new option doesn't seem relevant to this.

Reply:
The reason why I thought adding back that code would be redundant was that, the blockchain sync will now be run as the last thing. Also I expect that the sync status page to be displayed on special occasions, once everyone understands the purpose of the PR which is to make the user to never wait for the background syncing completion whenever possible, for them to use the web app. But I have had a changed of mind.

```go
	log.Infof("All ready, at height %d.", baseDBHeight)

	if cfg.SyncAndQuit {
		return nil
	}

	// collect the data now it was not collected earlier. Set up the monitors too.
	if explore.SyncStatus {
		if err := collectBlocksAfterSyncing(); err != nil {
			return err
		}

		explore.RetrieveUpdates()
	}

	// Deactivate displaying the sync status page after the db sync was
	// completed successfully.
	explore.SyncStatus = false

	// Wait for notification handlers to quit
	wg.Wait()

	return nil
```

I have looked at this code again and I can see that there is `wg.Wait()` to used wait for any of the running goroutines to send quit status.

It now makes sense to keep the `SyncAndQuit` implementation since we may not want to wait for the quit message to be sent just as you described.